tests/functional/adapter/custom_schema_tests/test_custom_schema.py

```diff
@@ -0,0 +1,82 @@
+import pytest
+from dbt.tests.util import (
+    check_relations_equal,
+    run_dbt
+)
+from tests.functional.adapter.custom_schema_tests.seeds import seed_queries
+
+_VIEW_1_SQL = """
+select * from {{ target.schema }}.seed
+""".lstrip()
+
+_VIEW_2_SQL = """
+{{ config(schema='custom') }}
+
+select * from {{ ref('view_1') }}
+""".lstrip()
+
+_VIEW_3_SQL = """
+{{ config(schema='test', materialized='table') }}
+
+
+with v1 as (
+
+    select * from {{ ref('view_1') }}
+
+),
+
+v2 as (
+
+    select * from {{ ref('view_2') }}
+
+),
+
+combined as (
+
+    select last_name from v1
+    union all
+    select last_name from v2
+
+)
+
+select
+    last_name,
+    count(*) as count
+
+from combined
+group by 1
+""".lstrip()
+
+
+class TestCustomProjectSchemaWithPrefix:
+    @pytest.fixture(scope="class")
+    def schema(self):
```

Comment:
What does this do? I suspect this is a noop

Reply:
Yep, removed